x2b_default_config.toml

```diff
@@ -0,0 +1,23 @@
+[sbatch-args]
+time = "04:00:00"
+mem = 16000
+nodes = 1
+cpus-per-task = 2
+job-name = "xnat2bids"
+output = "./logs/%J.txt"
+mail-user = "ford_mcdonald@brown.edu"
+mail-type = "ALL"
+
+[xnat2bids-args]
+session = "XNAT_E00114"
+bids_root = "/gpfs/data/bnc/shared/bids-export/"
+data_dir = "/gpfs/data/bnc"
```

Comment:
We don't need to pass this one... I just used to do it to facilitate understanding on the volume binding. But we can bind directly only what we use